application/src/main/java/org/thingsboard/server/actors/ruleChain/RuleNodeActorMessageProcessor.java

```diff
@@ -167,7 +168,7 @@ void onRuleChainToRuleNodeMsg(RuleChainToRuleNodeMsg msg) throws Exception {
 
     @Override
     public String getComponentName() {
-        return ruleNode.getName();
+        return ruleNode != null ? ruleNode.getName() : UNKNOWN_NAME;
```

Comment:
Introduce a method `getName(RuleNode ruleNode)`. Then, you may inline the UNKNOWN_NAME constant